### PR TITLE
Containerization V2

### PR DESCRIPTION
**Pull request by CathPag**

### CODE REVIEW

File: definitions/containerization.md [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
 
-Compared to VMs, container images are lightweight. The containerization process requires a file with a list of dependencies. This file can be version controlled the build process can be automated, allowing an organization to focus on other priorities while the automated processes take care of the build.
\ No newline at end of file
+### The problem it addresses 
+
+Before containers became prevalent, organizations relied on virtual machines (VMs) to orchestrate multiple applications on a single [bare-metal machine](https://github.com/cncf/glossary/blob/main/definitions/bare_metal_machine.md). VMs are significantly larger than containers and require a hypervisor to run. Due to the storage, backup, and transfer of these larger VM templates, creating the VM templates is also slow.
```
Comment (murillodigital) [inline definitions/containerization.md:14]: Another problem that containers address is that of mutability, I think it's relevant to highlight that containers are immutable, and the reliability and security guarantees that very relevant quality provide.

The problem being mutability and the ambiguity in terms of system state and reproducibility of that state, and the solution below immutability and predictable, known, versioned system states.

**Review by murillodigital — COMMENTED** [review]
Comment added.

File: definitions/containerization.md [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
+
+### How it helps
+
+Container images are lightweight (unlike VMs) and the containerization process requires a file with a list of dependencies. This file can be version controlled and the build process automated, allowing an organization to focus on other priorities while the automated processes take care of the build.
```
Comment (caniszczyk) [inline definitions/containerization.md:18]: I'd say "unlike traditional VMs" as microvms such as firecracker are getting pretty damn lightweight

**Comment by JasonMorgan** [comment]
Hey folks, I think all your review comments have been resolved. If we don't hear anything back I plan to merge it next Tuesday.

**Review by JasonMorgan — APPROVED** [review]

**Pull request merged**